### PR TITLE
HBASE-23552 Format Javadocs on ITBLL

### PR DESCRIPTION
**Pull request by ndimiduk**
We have this nice description in the java doc on ITBLL but it's
unformatted and thus illegible. Add some formatting so that it can be
read by humans.

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 34s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 26s |  master passed  |
| +1 :green_heart: |  compile  |   0m 30s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 19s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 37s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 17s |  master passed  |
| +0 :ok: |  spotbugs  |   4m 56s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m  0s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m  0s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 30s |  the patch passed  |
| -1 :x: |  checkstyle  |   0m 19s |  hbase-it: The patch generated 3 new + 35 unchanged - 0 fixed = 38 total (was 35)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   4m 46s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  15m 56s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   0m 15s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m  0s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 51s |  hbase-it in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 14s |  The patch does not generate ASF License warnings.  |
|  |   |  45m 52s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/920 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 38be6894c7a4 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-920/out/precommit/personality/provided.sh |
| git revision | master / 82e155eb26 |
| Default Java | 1.8.0_181 |
| checkstyle | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/1/artifact/out/diff-checkstyle-hbase-it.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/1/testReport/ |
| Max. process+thread count | 402 (vs. ulimit of 10000) |
| modules | C: hbase-it U: hbase-it |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/1/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

File: hbase-it/src/test/java/org/apache/hadoop/hbase/test/IntegrationTestBigLinkedList.java [hunk 1]
```diff
@@ -122,73 +122,100 @@
...
+ * by doing this. This test suite can also help find bugs that impact uptime
+ * and stability when run for days or weeks.
+ * </p>
+ * <p>
+ * This test suite consists the following
+ * <ul>
+ * <li>a few Java programs</li>
+ * <li>a little helper script to run the java programs</li>
+ * <li>a maven script to build it</li>
+ * </p>
```
Comment (HorizonNet) [inline hbase-it/src/test/java/org/apache/hadoop/hbase/test/IntegrationTestBigLinkedList.java]: A closing `ul` seems to be missing, leading to a wrong indentation afterwards in the output.

Comment (ndimiduk) [reply hbase-it/src/test/java/org/apache/hadoop/hbase/test/IntegrationTestBigLinkedList.java]: Yes you're right. Looks like checkstyle actually catches these issues, nice!

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 58s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
| +1 :green_heart: |  test4tests  |   0m  0s |  The patch appears to include 1 new or modified test files.  |
||| _ master Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m 31s |  master passed  |
| +1 :green_heart: |  compile  |   0m 29s |  master passed  |
| +1 :green_heart: |  checkstyle  |   0m 20s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 38s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   0m 16s |  master passed  |
| +0 :ok: |  spotbugs  |   4m 57s |  Used deprecated FindBugs config; considering switching to SpotBugs.  |
| +1 :green_heart: |  findbugs  |   0m  0s |  master passed  |
||| _ Patch Compile Tests _ |
| +1 :green_heart: |  mvninstall  |   5m  3s |  the patch passed  |
| +1 :green_heart: |  compile  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  javac  |   0m 30s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   0m 19s |  the patch passed  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  shadedjars  |   4m 37s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  hadoopcheck  |  15m 40s |  Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.1.2.  |
| +1 :green_heart: |  javadoc  |   0m 21s |  the patch passed  |
| +1 :green_heart: |  findbugs  |   0m  0s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  |   0m 52s |  hbase-it in the patch passed.  |
| +1 :green_heart: |  asflicense  |   0m 14s |  The patch does not generate ASF License warnings.  |
|  |   |  47m 12s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.5 Server=19.03.5 base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/920 |
| Optional Tests | dupname asflicense javac javadoc unit spotbugs findbugs shadedjars hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 785afe5eb990 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /home/jenkins/jenkins-slave/workspace/HBase-PreCommit-GitHub-PR_PR-920/out/precommit/personality/provided.sh |
| git revision | master / 82e155eb26 |
| Default Java | 1.8.0_181 |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/2/testReport/ |
| Max. process+thread count | 405 (vs. ulimit of 10000) |
| modules | C: hbase-it U: hbase-it |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-920/2/console |
| versions | git=2.11.0 maven=2018-06-17T18:33:14Z) |
| Powered by | Apache Yetus 0.11.1 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**